drafts/spice-up-your-server-side-rendered-apps-with-stimulus-and-turbolinks.md

```diff
@@ -0,0 +1,47 @@
+# 🌶 Spice up your server-side rendered apps with Stimulus and Turbolinks
+
+These days, SPA's (Single Page Applications) are an undeniable trend on the web. However, I personally think there are many use cases where a 'traditional' server-side rendered application still makes sense, for example for CRUD-style / admin applications or in general when your application doesn't require highly interactive features.
```

Comment:
Would split this in multiple sentences. It feels very long ;)

Reply:
`. For example for CRUD-style applications, `